autoload/gitmessenger/blame.vim

```diff
@@ -269,9 +269,9 @@ function! s:blame__after_blame(git) dict abort
     let author_email = matchstr(stdout[2], '^author-mail \zs\S\+')
     let self.state.contents = [
         \   '',
-        \   ' History: #' . self.state.history_no(),
-        \   ' Commit: ' . hash,
-        \   ' Author: ' . author . ' ' . author_email,
+        \   ' History:   #' . self.state.history_no(),
+        \   ' Commit:    ' . hash,
+        \   ' Author:    ' . author . ' ' . author_email,
```

Comment:
How about putting only one space after `History:` header in order to keep the window width as small as possible?

```vim
        \   ' History: #' . self.state.history_no(),
        \   ' Commit:  ' . hash,
        \   ' Author:  ' . author . ' ' . author_email,
```

Since `Committer:` line or `Author:` line is often the longest in the window, it affects the window width. Smaller window is better because a float window size is limited by Neovim's screen size.

Reply:
The problem is that `Committer:` is the longest header, but it's not always visible. If we reduce the number of spaces, once `Committer:` is visible things will not be aligned, e.g.:

```
History: #0
Commit:  0bc22aea857cb399988124ecbaa03c229fb14847
Author:  Alphonse Mariya <alphonse.mariya@email.com>
Committer: John Doe <john.doe@email.org>
```

But I got it, and I agree that making popups wider than they are already is not a great solution. Let me think about this.

Reply:
ok, I did not think about `Committer:` header. Let me merge this PR at first and add more fixes.